sdk/core/System.ClientModel/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@
 
 ### Breaking Changes
 
+- **Source Breaking Change**: Updated `IJsonModel<T>.Create` and `IPersistableModel<T>.Create` method return types from `T` to `T?` to allow returning `null` when deserialization fails. This change only affects code with nullable reference types enabled. 
```

Comment:
It looks like there are merge conflicts in this file. Pull from main and resolve the merge conflicts.